### DISCUSSION TITLE
App needs an icon.

### DISCUSSION THREAD

**Discussion by cristianberneanu** [diffix/desktop#82] [Ideas]
@fjab Do we have a logo for Open Diffix?

**Comment by fjab** [comment]
Unfortunately not. Until we have, maybe simply use the initials?

@yoid2000 you were initially opposed to creating logo / CI by a professional. Has this changed, now that we're taking a more UI-centered approach?

**Comment by yoid2000** [comment]
I think the initials are fine. "EDA". (Assuming we go with "Easy Data Anonymization" as the name of the thing)

**Reply by fjab** [reply]
I think the name should be a separate discussion...

**Comment by yoid2000** [comment]
If the thing is actually popular, we can get a professional logo as a congratulations gift to ourselves!

**Comment by cristianberneanu** [comment]
I made a couple of non-professional icons. Please vote the one you like most.

--------------------------------------
Option 1:
![logo_text](https://user-images.githubusercontent.com/5346001/128473169-27815a50-8e3f-4c89-b163-9f92ecb9e9ba.png)

**Comment by cristianberneanu** [comment]
Option 2: 
![logo_folks](https://user-images.githubusercontent.com/5346001/128473449-2cef9600-3825-4987-acff-ad7f16afba6a.png)

**Comment by cristianberneanu** [comment]
Option 3: 
![logo_disguise](https://user-images.githubusercontent.com/5346001/128473479-76ee7ef5-5c33-4076-ba1a-df55ed67de5c.png)

**Reply by fjab** [reply]
Hahaha, this is great 😂

**Reply by yoid2000** [reply]
I like this one a lot.

**Reply by yoid2000** [reply]
Cristian did you make this one, or just find it somewhere?

**Reply by cristianberneanu** [reply]
I made it yesterday from scratch (so no copyright issues with it).

**Reply by yoid2000** [reply]
Fucking brilliant, IMHO

**Reply by cristianberneanu** [reply]
So I assume this is the winner, right? I will tweak it a bit more and add it to the project.

**Reply by fjab** [reply]
I think it's good as an app icon. It's grown on me :)
